client/onboarding/style.scss

```diff
@@ -27,10 +30,12 @@ body.wcpay-onboarding__body {
 					display: flex;
 					align-items: center;
 					padding: $gap-large;
+					color: #0073aaff;
 					font-size: 14px;
 
 					.gridicons-chevron-left {
 						margin-right: 2px;
+						fill: #0073aaff;
```

Comment:
Similar to the previous comment, this is not a valid HEX color.

Reply:
Same as above :)